### PR TITLE
(chore) replace deprecated autoprefixer-loader with postcss-loader

### PR DESCRIPTION
**Pull request by jonsadka**
Per docs, [`autoprefixer-loader is deprecated. Autoprefixer official page recommends using postcss-loader instead.`](https://github.com/passy/autoprefixer-loader)

Migrate with recommended `postcss-loader`

### CODE REVIEW

File: webpack/build.babel.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default {
         test: /\.scss$/,
         use: ExtractTextPlugin.extract({
           fallback: 'style-loader',
-          use: ['css-loader', 'sass-loader', 'autoprefixer-loader'],
+          use: ['css-loader', 'postcss-loader', 'sass-loader'],
```
Comment (jonsadka) [inline webpack/build.babel.js]: Order reversed per docs https://github.com/postcss/postcss-loader#config-cascade

**Comment by Apercu** [comment]
https://github.com/SIGSEV/minus/commit/758c874421cf5193e7a9fff26446d2ea66e82ee9#comments

**Comment by meriadec** [comment]
@Apercu the PR don't bring back precss instead of sass, dont worry! :) Only different way to load autoprefixer.

That made apart, I think that the `postcss.config.js` should be on the `webpack/` folder instead of the `src/styles/` one.

I'm currently doing a POC migrating our styling to [styled-components](https://www.styled-components.com/).

**Comment by meriadec** [comment]
oww, wasn't enough fast to comment before the deletion 😂

**Comment by jonsadka** [comment]
I'll reopen. Feel free to merge or close this depending on what you two decide

**Comment by Apercu** [comment]
Sorry I just read the title and wanted to simply link to the related commit. Would you mind switching to the postcss-loader [plugin form](https://github.com/postcss/postcss-loader#plugins) rather than creating an additional file?

**Comment by jonsadka** [comment]
@Apercu updated

File: webpack/build.babel.js [hunk 2]
```diff
@@ -17,7 +17,19 @@ export default {
...
           fallback: 'style-loader',
-          use: ['css-loader', 'sass-loader', 'autoprefixer-loader'],
+          use: [
+            'css-loader',
+            {
+              loader: 'postcss-loader',
+              options: {
+                ident: 'postcss',
+                plugins: (loader) => [
+                  require('autoprefixer')()
```
Comment (Apercu) [inline webpack/build.babel.js]: should probably need to add it as a devDep too

Comment (jonsadka) [reply webpack/build.babel.js]: I added it in `webpack/dev.js`. Is that what you meant?

**Comment by Apercu** [comment]
Nice thanks!

**Pull request merged**